lib/t/accounts.ex

```diff
@@ -488,22 +505,37 @@ defmodule T.Accounts do
     end)
     |> Repo.transaction()
     |> case do
-      {:ok, %{profile: profile}} -> {:ok, profile}
-      {:error, :profile, %Ecto.Changeset{} = changeset, _changes} -> {:error, changeset}
+      {:ok, changes} ->
+        case changes do
+          %{profile: profile, gender_preferences: genders} ->
+            {:ok, %Profile{profile | gender_preference: genders, hidden?: false}}
+
+          %{profile: profile} ->
+            {:ok, %Profile{profile | hidden?: false}}
```

Comment:
why `hidden?: false`? why is this clause necessary?

Reply:
it was there before, not sure why as well (maybe_unhide should probably be used here)

Reply:
it wasn't

Reply:
<img width="969" alt="Screenshot 2021-11-13 at 18 49 33" src="https://user-images.githubusercontent.com/67764432/141650200-264c5bcd-24ef-4f1f-9b06-37cc91d58a19.png">

Reply:
right, my bad